src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -1,54 +1,70 @@
 // @debug $code-editor; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$code-editor} {
+:where(:root), :where(.#{$code-editor}) {
   // controls
-  --#{$code-editor}__controls--c-button--m-control--Color: var(--#{$pf-global}--Color--200);
-  --#{$code-editor}__controls--c-button--m-control--hover--Color: var(--#{$pf-global}--Color--100);
-  --#{$code-editor}__controls--c-button--m-control--focus--Color: var(--#{$pf-global}--Color--100);
-  --#{$code-editor}__controls--c-button--m-control--disabled--after--BorderBottomColor: var(--#{$pf-global}--BorderColor--100);
-
-  // Header
-  --#{$code-editor}__header--before--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$code-editor}__header--before--BorderBottomColor: var(--#{$pf-global}--BorderColor--100);
+  --#{$code-editor}__controls--PaddingInlineStart: var(--pf-t--global--spacer--sm);
+  --#{$code-editor}__controls--Gap: var(--pf-t--global--spacer--xs);
+
+  // header
+  --#{$code-editor}__header--before--BorderBottomWidth: var(--pf-t--global--border--width--control--default);
+  --#{$code-editor}__header--before--BorderBottomColor: var(--pf-t--global--border--color--default);
+  --#{$code-editor}__header--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
+  --#{$code-editor}__header--m-plain--BackgroundColor: var(--pf-t--global--background--color--primary--default);
```

Comment:
Nit: since these are being applied to the header-content element, we could rename the vars